tests/pytest_tests/system_tests/conftest.py

```diff
@@ -475,116 +476,14 @@ def pytest_addoption(parser):
-    """Check if wandb server is healthy.
-
-    :param base_url:
-    :param num_retries:
-    :param sleep_time:
-    :return:
-    """
-    for _ in range(num_retries):
-        try:
-            response = requests.get(urllib.parse.urljoin(base_url, endpoint))
-            if response.status_code == 200:
-                return True
-            time.sleep(sleep_time)
-        except requests.exceptions.ConnectionError:
-            time.sleep(sleep_time)
-    return False
-
-
-def spin_wandb_server():
-    def spin_wandb_server_fn(settings: WandbServerSettings) -> bool:
-        base_url = settings.base_url
-        app_health_endpoint = "healthz"
-        fixture_url = base_url.replace(
-            settings.local_base_port, settings.fixture_service_port
-        )
-        fixture_health_endpoint = "health"
-
-        if settings.wandb_server_use_existing:
-            return check_server_health(base_url=base_url, endpoint=app_health_endpoint)
-
-        if not check_server_health(base_url, app_health_endpoint):
-            command = [
-                "docker",
-                "run",
-                "--pull",
-                settings.wandb_server_pull,
-                "--rm",
-                "-v",
-                f"{settings.volume}:/vol",
-                "-p",
-                f"{settings.local_base_port}:{settings.internal_local_base_port}",
-                "-p",
-                f"{settings.services_api_port}:{settings.internal_local_services_api_port}",
-                "-p",
-                f"{settings.fixture_service_port}:{settings.internal_fixture_service_port}",
-                "-e",
-                "WANDB_ENABLE_TEST_CONTAINER=true",
-                "--name",
-                settings.name,
-                "--platform",
-                "linux/amd64",
-                f"{settings.wandb_server_image_registry}/{settings.wandb_server_image_repository}:{settings.wandb_server_tag}",
-            ]
-            subprocess.Popen(command)
-            # wait for the server to start
-            server_is_up = check_server_health(
-                base_url=base_url, endpoint=app_health_endpoint, num_retries=30
-            )
-            if not server_is_up:
-                return False
-            # check that the fixture service is accessible
-            return check_server_health(
-                base_url=fixture_url,
-                endpoint=fixture_health_endpoint,
-                num_retries=30,
-            )
-
-        return check_server_health(
-            base_url=fixture_url, endpoint=fixture_health_endpoint, num_retries=10
-        )
-
-    return spin_wandb_server_fn
-
-
 def pytest_configure(config):
     print("Running tests with wandb version:", wandb.__version__)
     print("Configuring wandb server...")
 
     settings = WandbServerSettings(
         name=DEFAULT_SERVER_CONTAINER_NAME,
         volume=DEFAULT_SERVER_VOLUME,
+        url=os.getenv("WANDB_TEST_SERVER_URL", DEFAULT_SERVER_URL),
```

Comment:
where is `WANDB_TEST_SERVER_URL` defined?

Reply:
thanks for catching; removed now that we are using docker `name` instead of port remapping